### ISSUE TITLE
Value range of String of Servitude doesn't search correctly

### ISSUE THREAD

**Issue by caddph** [SnosMe/awakened-poe-trade#1468]
String of Servitude triples the implicit values (normal range is 20-30; actual values are 60-90). When looking up via Awakened POE trade, the slider only selects between 20 and 30, but it doesn't search the listed trades by base value; it searches instead by total value.

So when you select 30 in the slider, the top trades that show are actually showing the bottom roll trades listed (e.g., when you open trade site, it's searching min value of 30, and the 60% belts are what are showing up).

TL;DR the slider for this belt should range between 60-90 not 20-30 (or it should search by 3x-ing the slider). Not sure if this affects other multiplied stat items.

**Comment by caddph** [comment]
Providing example screenshots; 1st image is Awakened POE; 2nd is when you click to jump to trade site; 3rd is manually searching for maxroll on trade site.
![01 AwakeSoS](https://github.com/user-attachments/assets/d06e3ec7-d1ae-4f3b-be04-e484766b9163)
![02 SoS trade site](https://github.com/user-attachments/assets/7d26f419-90a3-4e06-aade-41c7255399c6)
![03 SoS trade site - manual max roll](https://github.com/user-attachments/assets/20cd0f37-6aa0-460a-9ab0-9b01fbd2f684)

**Comment by mrooney** [comment]
I just ran into this and can confirm, I had two perfect 6% increased attribute rolls that were worth about a divine as 18%/18%, but the tool searched for 6% and so it showed the result as being worth a couple chaos. Luckily I verified on search but it would be great to fix this by tripling the search results :)

**Comment by caddph** [comment]
This is still unaddressed and continues to mislead players to list items far lower than their value on the market place.
